doc/user-guides/secure-protect-connect-single-multi-cluster.md

```diff
@@ -326,30 +335,31 @@ kubectl apply -f - <<EOF
 apiVersion: kuadrant.io/v1
 kind: DNSPolicy
 metadata:
-  name: ${gatewayName}-dnspolicy
-  namespace: ${gatewayNS}
+  name: ${GATEWAY_NAME}-dnspolicy
+  namespace: ${GATEWAY_NS}
 spec:
   loadBalancing:
     defaultGeo: true
-    geo: US
+    geo: GEO-US
```

Comment:
This was my fault. This is not a continent code. So this should be something like `GEO-NA`